src/util/nativeStorageHelper.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable fp/no-class */
```

Comment:
We would prefer to avoid classes on this project when possible. It's okay this time.